docs/source/usage/plugins/radiation.rst

```diff
@@ -65,7 +65,7 @@ namespace                     Description
 Observation directions
 """"""""""""""""""""""
 
-The number of observation directions `N_theta` is defined in :ref:`radiation.param <usage-params-plugins>`, but the distribution of observation directions is given in :ref:`radiationObserver.param.param <usage-params-plugins>`)
+The number of observation directions `N_theta` is defined in :ref:`radiation.param <usage-params-plugins>`, but the distribution of observation directions is given in :ref:`radiationObserver.param <usage-params-plugins>`)
```

Comment:
this must be double-ticked `` (`N_theta`) if you want this to be verbatim (otherwise it will be empathized as italic).